Source/Runtime/Events/EventStore.proto

```diff
@@ -28,6 +29,20 @@ message FetchForAggregateRequest {
     services.CallRequestContext callContext = 1;
     Aggregate aggregate = 2;
```

Comment:
```suggestion
    Aggregate aggregate = 2;
    option deprecated = true;
```

It seems like it might be possible to also deprecate `message`'s, but I'm not sure that it does anything interesting for the generated code though:
https://github.com/protocolbuffers/protobuf/issues/1734#issuecomment-234681729

It also mentions that you __can__ add it to the `rpc`'s but but it doesn't do anything (in 2019 at least):
https://github.com/protocolbuffers/protobuf/issues/1734#issuecomment-484464242

Just a cool thing I found that could make the deprecation even clearer. We are the only consumers though so w/e